### PR TITLE
updated reagent deps

### PR DESCRIPTION
**Pull request by cliffcaseyyet**
had some old reagent deps

### CODE REVIEW

**Review by milt — APPROVED** [review]
Well that at least looked easy.

File: deps.edn [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {:paths ["src/cljs"]
- :deps  {org.clojure/clojurescript {:mvn/version "1.10.520"}
-         reagent                   {:mvn/version "0.8.1"}
+ :deps  {org.clojure/clojurescript {:mvn/version "1.10.773"}
+         reagent                   {:mvn/version "1.1.0"}
          cljsjs/codemirror         {:mvn/version "5.44.0-1"}}}
```
Comment (milt) [inline deps.edn:4]: Is there a newer cljsjs package for cm?

Comment (cliffcaseyyet) [reply deps.edn:4]: @milt there is not :( It's the main thing i wanted to do but there is definitely not.

Comment (milt) [reply deps.edn:4]: womp womp

**Pull request merged**